app/views/users/new.html.erb

```diff
@@ -1 +1,14 @@
-<p>Favor fazer o login com sua conta do twitter</p>
+<div class="wrapper welcome-page-section">
+  <p class="mbl"><%= Settings.brief_explanation %></p>
+  <p>Connect your Twitter account to auto-send messages at <%= Settings.company_name %> to encourage them to improve their service. You will be able to disconnect or stop sending messages any time.</p>
+</div>
+
+<div class="welcome-page-section">
+  <p>Please connect your Twitter account</p>
+
+  <div class="wrapper-sm">
+    <%= link_to '/auth/twitter', class: 'btn-twitter' do %>
+      <span class="btn-twitter-icon-wrapper"><%= render 'svg/twitter' %></span> <span class="btn-twitter-text">Sign in with your Twitter Account</span>
```

Comment:
When I translated this message, the text shrinked but not the button. Can we make it expand according to the text size? 
<img width="493" alt="screen shot 2017-10-25 at 4 18 32 pm" src="https://user-images.githubusercontent.com/412144/31990800-46aa0a76-b9a0-11e7-9179-12fcba5ef116.png">

Reply:
Another thing that debs pointed out and I also noticed right before she said it is: this twitter blue is vibrating a lot with the bright red in the background. Try to scroll the site a bit and you should notice it.

Idk how to improve that, maybe darkening the blue a bit? Or the red? 🤔

Reply:
> Can we make it expand according to the text size?
We would have to write some JavaScript to have the font-size change dynamically! Instead, I hope it's ok that I increased the font-size (while also making sure it looks good on smaller devices)

![screen shot 2017-10-28 at 11 49 32 am](https://user-images.githubusercontent.com/5567838/32137499-21453840-bbd6-11e7-9744-eee8ac2dddd9.png)

> vibrating a lot with the bright red

I can't reproduce this, but we can darken the blue! I just thought people would recognize the twitter button more quickly if it was the brand color. Up to you!

Reply:
> We would have to write some JavaScript to have the font-size change dynamically!

Wouldn't a `display: inline-block` suffice? 🤔 

> I can't reproduce this

We could see it on my macbook monitor and also on her alienware notebook monitor, both with screen brightness at max and we don't use an app like those that changes the temperature of the monitor. Do you use one of those or usually use your monitor with decreased brightness?